teensy4/IntervalTimer.cpp

```diff
@@ -95,16 +94,23 @@ void IntervalTimer::end() {
 //FASTRUN
 static void pit_isr()
 {
+#if 0
 	for (int i=0; i < NUM_CHANNELS; i++) {
 		IMXRT_PIT_CHANNEL_t *channel = IMXRT_PIT_CHANNELS + i;
-		if (channel->TFLG) {
+		if (funct_table[0] && channel->TFLG) {
```

Comment:
Shouldn't that be `funct_table[i]` ?

Reply:
that's disabled.

Reply:
Sorry you are right of course. Looks like I'm getting too used to having not defined sections grayed out... 

I'll try it in a minute